landing-page/styles.css

```diff
@@ -0,0 +1,122 @@
+*{
+  margin:0;
+  padding:0;
+}
+body{
+  margin:0;
```

Comment:
You don't need this `margin` property on `body` because you've already defined a 0 margin on the universal selector (`*`), which `body` will inherit from.